modules/ecr/outputs.tf

```diff
@@ -1,10 +1,9 @@
-output "ecr_name" {
-  description = "The ECR name"
-  value       = var.pycaret_ecr_name
+output "repository" {
```

Comment:
Passing entire resources as outputs allows us to access more information about the resources created by the module. In 'streaming-data-platform', this was often essential to integrate the infrastructure created inside the module with that created outside the module.